src/main/java/Transcription/TranscriptionCache.java

```diff
@@ -0,0 +1,29 @@
+package Transcription;
+
+import Search.Searchable;
+
+public class TranscriptionCache {
+    private VoskAdapter speechToText;
+    private String filename;
+    private VoskProxy voskProxy;
+
+    public TranscriptionCache(String filename, VoskAdapter speechToText) {
+        this.filename = filename;
+        this.speechToText = speechToText;
+        this.voskProxy = new VoskProxy();
+    }
+
+    public Searchable getSearchable() {
```

Comment:
Changed to this pattern, since I think you mentioned that it is standard practice for a cache. However, I noticed it contradicts the suggestion that negative conditionals are unclear.